Makefile

```diff
@@ -12,6 +12,9 @@ sdk-utilities:
 python:
 combo-package:
 
+.NOTPARALLEL:
```

Comment:
What does this do?

Reply:
This disables the effect of my global MAKEFLAGS environment variable, which I have set to

export MAKEFLAGS=-j5

Without adding this to Makefile, the build would try to do things in parallel which needed to be serialized. I didn't spend time trying to figure out what additional dependencies would be needed to make parallel builds work correctly.

Reply:
Problem is come from `make install`, and there is fix in @issuehsu 's PR https://github.com/sifive/freedom-tools/pull/24/commits/7b23d03c5ffa2e4e738e43dca9271e13b99b7f16#diff-b67911656ef5d18c4ae36cb6741b7965R785

Reply:
thanks for creating a fix that doesn't disable parallel builds in the entire tree.